backend/battle/migrations/0001_initial.py

```diff
@@ -1,32 +1,20 @@
-# Generated by Django 3.2 on 2021-04-28 18:39
+# Generated by Django 3.2 on 2021-04-30 19:01
 
-from django.conf import settings
 from django.db import migrations, models
-import django.db.models.deletion
 
 
 class Migration(migrations.Migration):
 
     initial = True
 
     dependencies = [
-        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
     ]
 
     operations = [
         migrations.CreateModel(
             name='Battle',
             fields=[
-                ('id', models.AutoField(primary_key=True, serialize=False)),
```

Comment:
This will break your migrations in the production environment. I suggest you keep old migrations untouched and create new ones